src/main/java/codeu/model/store/basic/ConversationStore.java

```diff
@@ -55,49 +61,60 @@ public static ConversationStore getTestInstance(PersistentStorageAgent persisten
    */
   private PersistentStorageAgent persistentStorageAgent;
 
-  /** The in-memory list of Conversations. */
-  private List<Conversation> conversations;
+  /** The in-memory map of Conversations. */
+  private Map<UUID, Conversation> conversations;
+
+  /** A bimap from titles to IDs, so conversation IDs can be fetched from conversation titles
+   * quickly and vice versa.
+   */
+  private BiMap<String, UUID> titleToId;
```

Comment:
I just noticed that using a `BiMap` as opposed to just a `Map` is pretty silly, considering the user (and thus, the username) can be fetched from the ID already. I may remove this, along with the guava import. Thoughts?

Reply:
Up to you, probably best to keep the number of dependencies limited if it doesn't increase complexity to do so.

Reply:
Done removing the dependency.

Reply:
Done removing the dependency.